runners/flink/1.11/build.gradle

```diff
@@ -31,3 +35,19 @@ project.ext {
 
 // Load the main build script which contains all build logic.
 apply from: "$basePath/flink_runner.gradle"
+
+dependencies {
+  permitUnusedDeclared library.java.jackson_annotations
+  permitUnusedDeclared "org.apache.flink:flink-clients_2.11:$flink_clients_version"
```

Comment:
I don't understand how the flink client is unused. Please add a comment to explain each exclusion. We can try to ask people to understand.

Reply:
@kennknowles, using wfhartford plugin(https://github.com/wfhartford/gradle-dependency-analyze#configurations) for dependency check by running  "gradle analyzeDependencies --scan" gives result of of the unusedDeclared and usedUndeclared dependency list. The plugin says flink client is UnusedDeclared.

Reply:
This could be true because we often submit only against embedded clusters. However, if you look at `FlinkSubmissionTest`, it definitely utilizes the client dependency through the use of Clifrontend (part of flink-clients).